ansible/deploy.yml

```diff
@@ -111,6 +111,7 @@
       tags:
         - evo
       when: evo_services
+    - role: filebeat
```

Comment:
let's add an `elk-` prefix to group logically together with `elk-stack`?

Reply:
OK, I looked at how elastic name their products and they strongly discourage use of `ELK` now. Everything is called Elastic Stack, and Filebeat is part of Elastic Beats (which include many different types of collectors). So I have named it `elastic-stack` and `elastic-beats` to be consistent with their naming.

Reply:
Sounds good!

Reply:
it should be `elastic-filebeat`